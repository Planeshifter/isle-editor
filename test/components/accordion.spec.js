// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import Accordion from '@isle-project/components/accordion/main.js';


// TESTS //

describe( '<Accordion />', function test() {
	it( 'renders an accordion', () => {
		const accordion = renderer.create(
			<Accordion headers={[ 'First', 'Second' ]} active={0} >
				<div>First content</div>
				<div>Second content</div>
			</Accordion>
		).toJSON();
		expect( accordion ).toMatchSnapshot();
	});
});
