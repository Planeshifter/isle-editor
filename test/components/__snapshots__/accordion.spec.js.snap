// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Accordion /> renders an accordion 1`] = `
<div
  className="accordion "
  style={null}
/>
`;
